server/src/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -926,7 +926,7 @@ protected void checkIfPodIsDeletable(final long podId) {
                 dbName = "cloud";
             }
 
-            String selectSql = "SELECT * FROM `" + dbName + "`.`" + tableName + "` WHERE " + column + " = ?";
+            String selectSql = "SELECT * FROM `?`.`?` WHERE ? = ?";
 
             if(tableName.equals("vm_instance")) {
                 selectSql += " AND state != '" + VirtualMachine.State.Expunging.toString() + "' AND removed IS NULL";
```

Comment:
shouldn't be necessary :)
findbugs is smart enough to check that the returned string is static and cannot be polluted by user input
that's only not the case for other variables that it can't track the origin